clients/java/src/test/java/io/zeebe/client/ZeebeClientTest.java

```diff
@@ -125,4 +125,26 @@ public void shouldNotFailIfClosedTwice() {
     client.close();
     client.close();
   }
+
+  public void shouldCreateWorkflowInstances() throws URISyntaxException {
```

Comment:
I think we don't need this that as we already have https://github.com/zeebe-io/zeebe/commit/edcfbb02743566cd4b7ede7643082237db247f0f#diff-db5f7e2bc4e08c03e2a5dbd5c829402fR90